operations/host_spawn.go

```diff
@@ -199,7 +199,99 @@ func hostModify() cli.Command {
 	}
 }
 
-func hostlist() cli.Command {
+func hostStop() cli.Command {
+	const waitFlagName = "wait"
+	return cli.Command{
+		Name:  "stop",
+		Usage: "stop a running spawn host",
+		Flags: addHostFlag(
+			cli.BoolFlag{
+				Name:  waitFlagName,
+				Usage: "command will block until host stopped",
+			},
+		),
+		Before: mergeBeforeFuncs(setPlainLogger, requireHostFlag),
+		Action: func(c *cli.Context) error {
+			confPath := c.Parent().Parent().String(confFlagName)
+			hostID := c.String(hostFlagName)
+			wait := c.Bool(waitFlagName)
+
+			ctx, cancel := context.WithCancel(context.Background())
+			defer cancel()
+
+			conf, err := NewClientSettings(confPath)
+			if err != nil {
+				return errors.Wrap(err, "problem loading configuration")
+			}
+			client := conf.GetRestCommunicator(ctx)
+			defer client.Close()
+
+			if wait {
+				grip.Infof("Stopping host '%s'. This may take a few minutes...", hostID)
+			}
+
+			err = client.StopSpawnHost(ctx, hostID, wait)
+			if err != nil {
+				return err
```

Comment:
Should this be wrapped? And if wait should something be logged to the user? Same for start

Reply:
I think it's fine to leave it unwrapped, currently returns something like this:
> problem modifying host: 404 (Not Found): host with id '<ID>' not found

When the --wait option is specified, the user gets the following message right before the request is made:

> Stopping host '<ID>'. This may take a few minutes...